src/applications/vaos/utils/selectors.js

```diff
@@ -42,6 +41,16 @@ import {
   getVARFacilityId,
 } from '../services/appointment';
 
+export const selectIsCernerOnlyPatient = state =>
```

Comment:
This is the core change, creating VAOS versions of these selectors that check the appointments flag.